src/pyodide.js

```diff
@@ -177,7 +178,20 @@ var languagePluginLoader = new Promise((resolve, reject) => {
     return {};
   };
 
-  Module.locateFile = (path) => baseURL + path;
+  Module.locateFile = (path) => {
+    if ((window.hasOwnProperty('pyodide')) &&
+        (window.pyodide.hasOwnProperty('_toLoadPackages'))) {
```

Comment:
This should be unnecessary if you move it inside `_loadPackage` as well, since we'll already be in a state where this stuff is guaranteed to exist.